images/linux/scripts/installers/google-chrome.sh

```diff
@@ -25,3 +25,50 @@ fi
 # Document what was added to the image
 echo "Lastly, documenting what we added to the metadata file"
 DocumentInstalledItem "Google Chrome ($(google-chrome --version))"
+
+CHROME_VERSION=$(google-chrome --version | grep -Eo "([0-9]+\.?){4}" | cut -d "." -f 1,2,3)
+
+# Determine latest release of chromedriver
+# Compatibility of Google Chrome and Chromedriver: https://sites.google.com/a/chromium.org/chromedriver/downloads/version-selection
+LATEST_CHROMEDRIVER_VERSION=$(curl "https://chromedriver.storage.googleapis.com/LATEST_RELEASE_$CHROME_VERSION")
+
+# Download and unpack latest release of chromedriver
+echo "Downloading chromedriver v$LATEST_CHROMEDRIVER_VERSION..."
+wget "https://chromedriver.storage.googleapis.com/$LATEST_CHROMEDRIVER_VERSION/chromedriver_linux64.zip"
+unzip chromedriver_linux64.zip
+rm chromedriver_linux64.zip
+mv "chromedriver" "/usr/bin/chromedriver"
+chown root:root /usr/bin/chromedriver
+chmod +x /usr/bin/chromedriver
+
+# Run tests to determine that the chromedriver installed as expected
+echo "Testing to make sure that script performed as expected, and basic scenarios work"
+if ! command -v chromedriver; then
+    echo "chromedriver was not installed"
+    exit 1
+fi
+
+echo "Lastly, documenting what we added to the metadata file"
+DocumentInstalledItem "Chromedriver ($(chromedriver --version))"
+
+# Determine latest selenium standalone server version
+SELENIUM_LATEST_VERSION_URL=https://api.github.com/repos/SeleniumHQ/selenium/releases/latest
+SELENIUM_VERSION=$(curl $SELENIUM_LATEST_VERSION_URL | jq '.name' | tr -d '"' | cut -d ' ' -f 2)
+SELENIUM_VERSION_MAJOR_MINOR=$(echo $SELENIUM_VERSION | cut -d '.' -f 1,2)
+
+# Download selenium standalone server (hardcoded version 3.141.59)
+echo "Downloading selenium-server-standalone v$SELENIUM_VERSION..."
+SELENIUM_JAR_NAME=selenium-server-standalone-$SELENIUM_VERSION.jar
+wget https://selenium-release.storage.googleapis.com/$SELENIUM_VERSION_MAJOR_MINOR/$SELENIUM_JAR_NAME
+
+echo "Testing to make sure that script performed as expected, and basic scenarios work"
```

Comment:
does it makes sense to add note to these `echo`s (also on the line 45 for chromedriver) about what we are testing just to make it a bit clearer?
something like `Testing Selenium JAR downaloaded correctly`
Also it is not clear what are the basic scenarios we are testing here?

Reply:
This `echo` is used for each tool we install. I believe it would be inconsistent with the other ones

Reply:
Also, as you can see, there are logs, which tool we're installing on the lines 36 and 62. If something goes wrong, we'll be able to determine what tool caused the failure with these logs

Reply:
okay, thanks for the clarification.